src/plugins/intel_npu/src/plugin/npuw/util.cpp

```diff
@@ -450,10 +464,24 @@ inline void twrite_4b(ov::Tensor& t, uint8_t value, std::size_t r, std::size_t c
     }
 }
 
+inline void twrite_f16(ov::Tensor& t, uint16_t value, std::size_t r, std::size_t c, std::size_t COLS) {
+    uint16_t* tdata = static_cast<uint16_t*>(t.data());
+    uint16_t* trow = tdata + r * COLS;
+    uint16_t* telem = trow + c;
+    *telem = value;
+}
+
+inline void twrite_f32(ov::Tensor& t, float value, std::size_t r, std::size_t c, std::size_t COLS) {
+    float* tdata = static_cast<float*>(t.data());
+    float* trow = tdata + r * COLS;
+    float* telem = trow + c;
+    *telem = value;
+}
```

Comment:
This is a template too, but I doubt if you actually need it

Reply:
up. OK I see you need it for the ol' tranpose (which is not the generic permute)

Reply:
Done